packages/hash/shared/src/types.ts

```diff
@@ -1,393 +1,14 @@
-import { BaseUri, VersionedUri } from "@blockprotocol/type-system";
-import slugify from "slugify";
-import { systemUserShortname } from "@hashintel/hash-shared/environment";
-import { frontendUrl } from "./environment";
+// The types exported here are expected to be branded.
+// Perhaps some of these should be put into the subgraph stdlib instead?
 
-type SchemaKind = "data-type" | "property-type" | "entity-type";
+export type ownedById = string;
+export type AccountId = string;
 
-/** Slugify the title of a type */
-export const slugifyTypeTitle = (title: string): string =>
-  slugify(title, { lower: true });
+export type EntityId = string;
+export type EntityUuid = string;
+export type EntityEditionId = string;
```

Comment:
This is an object, shouldn't need branding. We do have an `EntityEditionIdString` type though 🤮  (still hate it and don't have a good idea of how to handle it)

Reply:
Should I omit it from here for now, then?

Reply:
Let's omit it for now